src/main/java/core/basesyntax/Figure.java

```
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements Behavior {
+    protected Color color;
+
+    @Override
+    public double getArea() {

```

Consider making this method `abstract` instead of providing a default implementation. Since every concrete figure must have its own way of calculating the area, an `abstract` method will enforce this contract at compile time. A default implementation returning `0` could hide a bug if a new figure subclass forgets to override it.